maskrcnn_benchmark/engine/trainer.py

```diff
@@ -55,6 +55,10 @@ def do_train(
     start_training_time = time.time()
     end = time.time()
     for iteration, (images, targets, _) in enumerate(data_loader, start_iter):
+        
+        if any(len(target) < 1 for target in targets):
+            logger.error(f"Iteration={iteration + 1} || Image Ids used for training {_} || targets Length={[len(target) for target in targets]}" )
```

Comment:
This is python-3 only.

But well, I suppose that with APEX we now have a hard-dependency on Python3 anyway